clang/include/clang/Basic/BuiltinsPPC.def

```diff
@@ -944,6 +944,9 @@ TARGET_BUILTIN(__builtin_pack_vector_int128, "V1LLLiULLiULLi", "", "vsx")
 // Set the floating point rounding mode
 BUILTIN(__builtin_setrnd, "di", "")
 
+// Barrier for instruction motion
+BUILTIN(__builtin_ppc_fence, "v", "")
```

Comment:
nit: maybe we can put this builtin around line 100, together with other XL Compatibility built-ins.